frontend/ui/data/optionsutil.lua

```diff
@@ -123,34 +152,46 @@ function optionsutil.showValues(configurable, option, prefix, document)
                                             current, value_current, default)
         end
     else
-        text = T(_("%1\n%2\nCurrent value: %3\nDefault value: %4"), name_text, help_text, current, default)
+        local unit
+        if is_size == 1 then
+            unit = G_reader_settings:nilOrTrue("metric_length") and "mm" or "\""
+        else
+            unit = "pt"
```

Comment:
May be here and in the callers, except of `1` and `2`, use `"pt"` or `"mm"` and `if size_type == "mm" elseif size_type == "pt"` (event if you end up using inch because of the preference).